monitor_ingest/actions/data_clean_csv_action.py

```diff
@@ -0,0 +1,102 @@
+import json
+from decimal import Decimal
+
+import numpy as np
+import pandas as pd
+import pandas_schema
+from pandas_schema import Column
+from pandas_schema.validation import CustomElementValidation
+from st2common.runners.base_action import Action
+
+__all__ = ["CleanCsvDataAction"]
+
+
+class CleanCsvDataAction(Action):
+    def __init__(self, config):
+        super(CleanCsvDataAction, self).__init__(config)
+        self._config = self.config
+        self._data_file_path = self._config.get('data_file_path', None)
+        self._json_schema_path = self._config.get('json_schema_path', None)
+
+        if not self._config:
+            raise ValueError('Missing config yaml')
+        if not self._data_file_path:
+            raise ValueError('Missing CSV data file path in config file')
+        if not self._json_schema_path:
+            raise ValueError('Missing JSON Schema data file path in config file')
+
+    def check_decimal(self, dec):
+        try:
+            Decimal(dec)
+        except ValueError:
+            return False
+        return True
+
+    def check_time_stamp(self, dt):
+        try:
+            pd.to_datetime(dt)
+        except ValueError:
+            return False
+        return True
+
+    def check_int(self, num):
+        try:
+            int(num)
+        except ValueError:
+            return False
+        return True
+
+    def run(self):
+        # define validation elements
+        print('1. Starting data Clean Action ..')
```

Comment:
Should replace prints with `self.logger.debug()`

Reply:
updated the changes for print